### ISSUE TITLE
Document granule "umm" and "meta" members

### ISSUE THREAD

**Issue by MattF-NSIDC** [nsidc/earthaccess#368]
Follow-up from #367.

Granules have `granule["umm"]` and `granule["meta"]` members that are not currently documented. What's the best way to document this? Expose them in a way that our autodoc gets it? Or write a how-to? Both?

**Comment by andypbarrett** [comment]
The UMM and Metadata components are/should be documented elsewhere.  If they are documented elsewhere, I don't think `earthaccess` needs to document what these metadata components are.  Maybe just link to other online resources.  However, we should expose/demonstrate how to access these.

Would something as simple as `granules["umm"].keys()` do the job.  Along with dome common use cases such as the xml example here.

**Comment by mfisher87** [comment]
>However, we should expose/demonstrate how to access these.

:100: Agreed we shouldn't document the structure of the data, just how to access the attributes from _earthaccess_!

**Comment by jessnicwelch** [comment]
> > However, we should expose/demonstrate how to access these.
> 
> 💯 Agreed we shouldn't document the structure of the data, just how to access the attributes from _earthaccess_!

I also agree... better to maintain only one source. If you know already, could you say where these members are used in the _earthaccess_ guide and a link to the source material?

**Comment by mfisher87** [comment]
I don't believe we have any documentation that references these attributes. Yet :)
